### ISSUE TITLE
Port Robots_Per_Team constant into python, and refactor all magic number 6s

### ISSUE THREAD

**Issue by jcarn** [RoboJackets/robocup-software#957]
Right now we just use "6" in a few places. Magic numbers are bad, and we should replace them. There is a constant called "Robots_Per_Team" in Constants.hpp that would be perfect for this, but it is in the C++ section of the code. Use the robocup-py.ccp stuff to get that constant into python, and then change all 6s to that constant

**Comment by jcarn** [comment]
Actually we could use len(main.our_robots()) if we wanted

**Comment by jgkamat** [comment]
`len(main.our_robots)` gives us the current number of robots we have (which we usually want) but the c++ constant gives us the max number of robots on a team (which is what a few plays, like stopped, need). I think we could make a function for `len(our_robots)` but we can't make it a constant since the number of robots we have might change in a match.

**Comment by longoodr** [comment]
@mmurley3 Are you still working on this?

**Comment by mmurley3** [comment]
Nah, I haven't had the time to make much progress on it. Feel free to tackle it if you want.

**Comment by Jason27chan** [comment]
@jcarn Which files should we be looking at?

**Comment by jcarn** [comment]
This was more of an intro issue I had in mind

**Comment by kylestach** [comment]
@tjones320 this is a must-have before competition this year.
